frame/session/benchmarking/src/lib.rs

```diff
@@ -27,16 +27,14 @@ use sp_runtime::traits::{One, StaticLookup, TrailingZeroInput};
 use sp_std::{prelude::*, vec};
 
 use frame_benchmarking::v1::benchmarks;
-use frame_support::{
-	codec::Decode,
-	traits::{Get, KeyOwnerProofSystem, OnInitialize},
-};
+use frame_support::traits::{Get, KeyOwnerProofSystem, OnInitialize};
 use frame_system::{pallet_prelude::BlockNumberFor, RawOrigin};
 use pallet_session::{historical::Pallet as Historical, Pallet as Session, *};
 use pallet_staking::{
 	benchmarking::create_validator_with_nominators, testing_utils::create_validators,
 	MaxNominationsOf, RewardDestination,
 };
+use sp_runtime::codec::Decode;
```

Comment:
why's `codec` re-exported from `sp_runtime`? I see it being used directly in other places

Reply:
Yeah, that is also a "forbidden" export.